lib/ui/views/home/home_viewmodel.dart

```diff
@@ -39,13 +39,13 @@ class HomeViewModel extends BaseViewModel {
   bool showUpdatableApps = false;
   List<PatchedApplication> patchedInstalledApps = [];
   List<PatchedApplication> patchedUpdatableApps = [];
-  String _managerVersion = '';
+  String? _latestManagerVersion = '';
 
   Future<void> initialize(BuildContext context) async {
-    _managerVersion = await AboutInfo.getInfo().then(
+    _latestManagerVersion = await AboutInfo.getInfo().then(
       (value) => value.keys.contains('version') ? value['version']! : '',
     );
-    _managerVersion = await _managerAPI.getCurrentManagerVersion();
+    _latestManagerVersion = await _managerAPI.getCurrentManagerVersion();
```

Comment:
This renaming does not make sense. Because the field represents the current version of the manager.

Reply:
This is already in the code base but why is this field assigned twice? The managerAPI class suggests (by name) that it does API stuff so why does it know about the current manager version?

Reply:
ah, on 48, i meant to change to getLatestManagerVersion()